src/components/Contributors.tsx

```diff
@@ -0,0 +1,167 @@
+import { Component } from 'react';
+import * as React from 'react';
+
+interface IContributorsState {
+    key: number;
+    photo: string;
+    githubPage: string;
+    githubName: string;
+    commits: number;
+}
+
+interface IReposState {
+    key: number;
+    name: string;
+    description: string;
+    link: string;
+}
+
+interface IContributorsArrayState {
+    ignoreRepos: string[];
+    ignoreContributors: string[];
+    repos: IReposState[];
+    contributors: IContributorsState[];
+}
+
+class Contributors extends Component<{}, IContributorsArrayState> {
+    
+    constructor(props: any) {
+        super(props);
+    
+        this.state = {
+            ignoreRepos: ["sicp", "sharedb-ace-backend", "assessments", "tools", "source-academy2"],
+            ignoreContributors: ["dependabot[bot]", "dependabot-preview[bot]"],
+            repos: [],
+            contributors: []
+        };
+    }
+
+    public componentDidMount() {
+
+        fetch('https://api.github.com/orgs/source-academy/repos')
+        .then(results => {
+            return results.json();
+        })
+        .then(repos => {
+            const { ignoreRepos } = this.state;
+            const contributorLinks = repos
+                .filter((repo: any) => {
+                    for (const repoName of ignoreRepos) {
```

Comment:
can be simplified, and try to avoid using the `any` term

```typescript
const contributorLinks = repos
  .filter(repo => !(repo.name in ignoreRepos)
```

Reply:
That's not how the `in` operator works. Use `array.includes(item)` to check if array contains the item.

Reply:
Yep my bad, i was mistaken

Reply:
I have simplified the code within filter in my latest commit.